packages/mathjax-extension/src/index.ts

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+/**
+ * @packageDocumentation
+ * @module mathjax-extension
+ */
+
+import {
+  JupyterFrontEnd,
+  JupyterFrontEndPlugin
+} from '@jupyterlab/application';
+
+import { ILatexTypesetter } from '@jupyterlab/rendermime';
+
+import type { MathDocument } from 'mathjax-full/js/core/MathDocument';
+
+enum CommandIDs {
+  copy = 'mathjax:clipboard',
+  scale = 'mathjax:scale'
+}
```

Comment:
Let's be consistent with the rest of the code

```suggestion
namespace CommandIDs {
  /**
   * Copy MathJax to clipboard
   */
  export const copy = 'mathjax:clipboard';
  /**
   * Scale MathJax elements.
   */
  export const scale = 'mathjax:scale';
}
```

Reply:
Fixed in latest revision.